src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
```

Comment:
Try not to use such comments. There should be a description or something that describes what for, version, author, time.

[More info](https://mate-academy.github.io/style-guides/java/java.html#s7.1-javadoc-formatting:~:text=7%20Javadoc-,7.1%20Formatting,-7.1.1%20General%20form)